melos.yaml

```diff
@@ -15,6 +15,10 @@ command:
     # branch: master
     # Additionally build a changelog at the root of the workspace.
     workspaceChangelog: true
+    hooks:
+      preCommit: |
+        dart run scripts/generate_version.dart && \
```

Comment:
Should it be generate_vertexai_version ?

Reply:
good catch, I think I went too wild with `cmd + z` 👍

Reply:
"vertexai" is the correct naming if we're being pedantic 😆